playground/nextjs-app-router/app/mint/MintComponent.tsx

```diff
@@ -28,7 +28,7 @@ export default function MintComponent() {
         <NFTCreator className="-mt-1 pt-0" />
         <NFTMedia />
         <NFTCollectionTitle />
-        <NFTMintButton className="[&_button]:bg-[#0052ff] [&_button]:hover:bg-[#014ceb] [&_button]:active:bg-[#0148dc] [&_button]:disabled:bg-[#80a8ff]" />
+        <NFTMintButton className="[&_button]:bg-[#0052ff] [&_button:hover]:bg-[#014ceb] [&_button:active]:bg-[#0148dc] [&_button:disabled]:bg-[#80a8ff]" />
```

Comment:
@cpcramer this doesn't look right to me. 

Here's the (tailwind docs)[https://tailwindcss.com/docs/hover-focus-and-other-states#using-arbitrary-variants]:
<img width="385" alt="image" src="https://github.com/user-attachments/assets/7dc12235-7fe6-42be-91b7-7b2a5641ba4e" />

IIUC, the brackets go around the arbitrary variant, but do not include the active state.